frontend/src/components/grantaccess/form/UserNameInput.vue

```diff
@@ -90,6 +90,7 @@ watch(
                         >{{ helperText }}</small
                     >
                     <ErrorMessage
+                        data-testid="test"
```

Comment:
Hi Jota, just wondering if we need this line? thanks!

Reply:
Hello, this line should have been cleaned up